libs/openpgm/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openpgm
+PKG_VERSION:=5.3.128
+PKG_RELEASE:=1
+
+PKG_SOURCE:=release-5-3-128.tar.gz
+PKG_SOURCE_URL:=https://github.com/steve-o/openpgm/archive
```

Comment:
Use codeload

Reply:
I tried to download source tarball from codeload, but the downloaded file names can only be `master` or `commit-ID`, without any `openpgm` prefix. For example, the latest release tarball has a corresponding codeload [URL](https://codeload.github.com/steve-o/openpgm/tar.gz/cd3456a5c437bb542d9af2a37778ec7941dc05a5), the saved file name is the `commit-ID`. I think setting PKG_SOURCE_PROTO to git, might be able to fetch a source with openpgm prefix.

Reply:
I see the issue. The URL is https://codeload.github.com/steve-o/openpgm/tar.gz/release-5-3-128  but that does not match the version.

Reply:
I've updated the PR again, using the codeload URL, the tarball is named as openpgm-release-5-3-128.tar.gz by download.pl, via setting variable `PKG_SOURCE_URL_FILE`. Is it necessary to change PKG_VERSION from `5.3.128` to `5-3-128`?